.github/workflows/linter.yml

```diff
@@ -1,7 +1,11 @@
-name: CI
+name: linter
```

Comment:
Let's keep the name unedited.

Because, we will have empty sections in the Actions dashboard otherwise.